Firestore/core/src/firebase/firestore/local/local_documents_view.h

```diff
@@ -103,6 +103,15 @@ class LocalDocumentsView {
   /** Queries the remote documents and overlays mutations. */
   model::DocumentMap GetDocumentsMatchingCollectionQuery(FSTQuery* query);
 
+  /**
+   * Find all `PatchMutation`s, and if their base documents are not in
+   * `existingDocs` add them to the returning `DocumentMap`. The returned map
+   * also has every document in `existingDocs`.
+   */
+  model::DocumentMap AddMissingBaseDocument(
+      const std::vector<FSTMutationBatch*>& matchingBatches,
```

Comment:
Nit: C++ uses `snake_case` for variable names (this can be a little confusing because Objective-C uses `camelCase`). We consider files under `core/src` to be C++ and files under `Source` to be Objective-C, so this file should follow C++ style rules. Some of the surrounding code gets this wrong (no need to fix anything you didn't add in this PR).

Applies throughout.

Reply:
Done.

Reply:
Done.